builtin/settingtypes.txt

```diff
@@ -265,8 +265,8 @@ undersampling (Undersampling) int 1 1 8
 
 [**Graphics Effects]
 
-#    Makes all liquids opaque
-opaque_water (Opaque liquids) bool false
+#    Allows liquids to be translucent
+translucent_water (Enable translucent liquids) bool true
```

Comment:
> Would you like me to make it "translucent_liquid" ?

I think this would be good:

```suggestion
#    Allows liquids to be translucent.
translucent_liquids (Translucent liquids) bool true
```

Reply:
I just addedd that am and trynna squash commits rn, i have rly slow internet at the moment